pages/education.py

```diff
@@ -84,64 +120,127 @@ def render_education_content(tab):
                         )
                     )
                 ),
+                dbc.Row(
+                    [
+                        # How has total expenditure changed over time?
+                        dbc.Col(
+                            [
+                                html.P(
+                                    id="education-narrative",
+                                    children="loading...",
+                                    style={"height": "100px", "margin": "10px"},
```

Comment:
maybe because of the style height restriction, when the window is resized slightly, the ending part of the narrative text seems to be hidden below the plot (from the same column). I think the height parameter might be the reason -- is there any specific reason for including this?

Reply:
Thanks that's a good catch!
It was my attempt to align the plots horizontally. Without these styling, the layout looks a bit ugly.
<img width="1163" alt="image" src="https://github.com/user-attachments/assets/3384d67a-fd54-4d57-ab87-5bfd41bc0eac">


But as you have already pointed out, it was not the best approach. Let me try something else!

Reply:
Took a different approach for aligning the plots.
<img width="1491" alt="image" src="https://github.com/user-attachments/assets/de2c4f81-0198-4110-8139-e2058356d125">

Reply:
This looks good! It seems like you have a row with two columns and then within the columns we have the narrative and then the plot. With this alignment, it seems like the narratives are potentially misaligned. Instead is it possible to have two rows each with two columns; the first row contains the narratives in each of the columns and then second row each of the columns contains the plots. 

Minor point -- please ignore if not currently relevant